src/components/Modals/Onboarding/OnboardingBasicSingleDropdown.vue

```diff
@@ -91,10 +91,10 @@ export default defineComponent({
       }
 
       // scroll the bottom of the graduation year dropdown to scrollBottomToIndex
```

Comment:
Nit: update this comment so it says `scrollBottomToElement` instead of `scrollBottomToIndex`